control-plane/config/eventing-kafka-broker/200-controller/500-controller.yaml

```diff
@@ -78,6 +78,15 @@ spec:
             - name: SOURCE_CONTRACT_CONFIG_MAP_NAME
               value: kafka-source-sources
 
+            - name: BROKER_DATA_PLANE_CONFIG_CONFIG_MAP_NAME
+              value: config-kafka-broker-data-plane
+            - name: SOURCE_DATA_PLANE_CONFIG_CONFIG_MAP_NAME
+              value: config-kafka-source-data-plane
+            - name: SINK_DATA_PLANE_CONFIG_CONFIG_MAP_NAME
+              value: config-kafka-source-data-plane
+            - name: CHANNEL_DATA_PLANE_CONFIG_CONFIG_MAP_NAME
+              value: config-kafka-source-data-plane
+
```

Comment:
I just realized that we need to make adjustments to this approach when we have dataplane configurations per namespace in the future.

With this PR, we always read the dataplaneConfigmap from the system namespace right?

Reply:
I'm not sure this is just the name, I think we can still require to have this exact name but in their namespace.

afaik, it's not a requirement that we have so far.

Reply:
however, "delivery profiles" feature is the way to go IMHO https://github.com/knative-sandbox/eventing-kafka-broker/issues/1412